stock_picking_partner_invoice_address/views/stock_picking_view.xml

```diff
@@ -0,0 +1,44 @@
+<odoo>
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="name">stock.picking.form</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='partner_id']" position="after">
+                <field name="partner_invoice_id"/>
+                <field name="same_invoiced_id"/>
+            </xpath>
+        </field>
+    </record>
+    <record id="vpicktree" model="ir.ui.view">
+        <field name="name">stock.picking.tree</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.vpicktree" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='partner_id']" position="after">
+                <field name="partner_invoice_id"/>
```

Comment:
Note that this field will be showing regardless of the operation type.  If we want to adjust the tree view based on the operation type, we need to do something like this: https://github.dev/qrtl/hls-custom/blob/e7528ff083ddf7198e53f4160428768014ac103a/stock_picking_delivery_due_date/models/stock_picking.py#L33-L51

Reply:
This part is currently not working, so I have commented out.